### ISSUE TITLE
Update wireframes to include spec for messaging

### ISSUE THREAD

**Issue by cazinc** [CDLUC3/dmsp_frontend_prototype#126]
Update existing wireframes to ensure that a sufficient spec is in place for messaging, e.g. notices on a page, form validation errors, etc.

**Comment by fraserclark** [comment]
We have created page trying to show possible styling of basic status messages

https://www.loom.com/share/4aa338636b3a45289456cc1b7817b153

Figma file referenced

https://www.figma.com/proto/RRtoRLyr5Spf0KiyRfEclt/DMP-Tool-Wireframes?node-id=2295-1510&node-type=frame&t=eX1fgwI5TfzbQsgl-0&scaling=min-zoom&content-scaling=fixed&page-id=736%3A1383&starting-point-node-id=2295%3A1510&show-proto-sidebar=1

**Comment by jupiter007** [comment]
Thank you Fraser. I like the message styling.

The only question I have is whether we will also want to include placement, and possibly different styling, for flash messages down the line.

Juliet Shin
University of California, Office of the President
1100 Broadway, 10th Flr, 10009B, Oakland, CA 94607
***@***.******@***.***>



From: Fraser Clark ***@***.***>
Date: Friday, October 4, 2024 at 6:32 AM
To: CDLUC3/dmsp_frontend_prototype ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [CDLUC3/dmsp_frontend_prototype] Update wireframes to include spec for messaging (Issue #126)

CAUTION: EXTERNAL EMAIL

We have created page trying to show possible styling of basic status messages

https://www.loom.com/share/4aa338636b3a45289456cc1b7817b153

Figma file referenced

https://www.figma.com/proto/RRtoRLyr5Spf0KiyRfEclt/DMP-Tool-Wireframes?node-id=2295-1510&node-type=frame&t=eX1fgwI5TfzbQsgl-0&scaling=min-zoom&content-scaling=fixed&page-id=736%3A1383&starting-point-node-id=2295%3A1510&show-proto-sidebar=1

**Comment by fraserclark** [comment]
@jupiter007 What are you thinking? Happy to do more on it.

I know Andre suggested "toast" messages too - I just hadn't fully done the research on the accessibility of toast messages

**Comment by jupiter007** [comment]
@fraserclark We could use a similar look to the message levels you proposed, but maybe outside of the white box container/page? We might want to distinguish page-level messaging from global messaging after a user is redirected, like when a user successfully logs in and is redirected to the home page, or users are redirected due to an error. Here's a short discussion about it: https://justinskord.medium.com/implementing-flash-messages-to-your-website-64775b7552e2.

**Comment by fraserclark** [comment]
Yeah we were thinking was we'd just have them same styling above the white box for global

I know I mostly did the "form" as it was a good "real world example" the placement could be wherever. The difference being as you said "global" vs more "local" e.g the problem/issue is related to the form vs a more global message.

**Comment by jupiter007** [comment]
Ok thanks @fraserclark. Sounds good to me.

**Comment by bofstein** [comment]
I'm not clear from the conversation if Fraser you are making updates to include that "global messaging" or if you're saying it's jsut like this and not needed.  But it would be nice to add it in for completeness, then I think we can mark this done since it looks good.  It's just another level that's more clearly the whole site and not just this page.  Here it is on the current site, informational warning.

![Image](https://github.com/user-attachments/assets/4a54fedf-dff0-44af-b2d7-dc8a791354ba)

**Comment by fraserclark** [comment]
Attached example of the "global" site notice; 

![Image](https://github.com/user-attachments/assets/ef1de992-f156-4311-b077-c950b2c68933)

**Comment by bofstein** [comment]
Awesome, thanks

**Comment by cazinc** [comment]
@bofstein @jupiter007 are you happy to close this one? Or is there anything else you'd like to see here for reference?

**Comment by jupiter007** [comment]
I'm ok with closing this.
